views/admin/user/all_users.php

```diff
@@ -15,44 +15,39 @@
       echo "<br>";    
         $stmt=$conn->prepare("SELECT * FROM `user`");
         $stmt->execute();
-        $result=$stmt->fetchAll();
-            
+        $users=$stmt->fetchAll();
+       if(count($users)>0){     
          echo "<table class='table  table-hover'>
          <thead  >
-           <tr>
+           <tr class='text-light bg-dark'>
              <th scope=col>ID</th>
-             <th scope=col>Room_id</th>
              <th scope=col>Name</th>
+             <th scope=col>Room_id</th>
              <th scope=col>Email</th>
-             <th scope=col>Password</th>
              <th scope=col>Exetra_Info</th>
              <th scope=col>Avatar</th>
              <th scope=col>Actions</th>
            </tr>
          </thead>";
-     foreach($result as $resul){
-     echo "<tbody> <tr> <td> ".$resul['user_id']. "</td>".
-      "<td> ".$resul['room_id']. "</td>".
-      "<td> ".$resul['name']. "</td>".
-      "<td> ".$resul['email']. "</td>".
-      "<td> ".$resul['password']. "</td>".
-      "<td> ".$resul['extra_info']. "</td>".
-     "<td>"."<img src=".'../../../images/users/'.$resul['avatar']." width=150 height=100 style='border-radius:50%;'>"."</td>".
-     "<td>" ."<button class='btn btn-danger'>Delete</button>
-     <button class='btn btn-primary'>Edit</button>" ."</td>".
-     
-     "</tr> </tbody>";
-     }
+         foreach($users as $user){
+          echo "<tr id='".$user['user_id']."'><th scope='row'>".$user['user_id']."</th>".              
+          "<td> ".$user['name']."</td>".
+          "<td> ".$user['room_id']."</td>".
+          "<td> ".$user['email']."</td>".
+          "<td> ".$user['extra_info']."</td>".
```

Comment:
You must not show data that was entered by the user to the screen before sanitizing it.